elife/config/lib-systemd-system-selenium.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description="Selenium server"
+
+[Install]
+WantedBy=multi-user.target
+
+[Service]
```

Comment:
any good resource for Systemd configuration? As an equivalent to http://upstart.ubuntu.com/cookbook/